### PR TITLE
Update fleets/app in develop section

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Change-type: patch
Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipul@balena.io>

---

> *Please make sure to read the [CONTRIBUTING](https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md) document before opening the PR for relevant information on contributing to the documentation. Thanks!*

### CODE REVIEW

File: pages/learn/develop/container-contracts.md [hunk 1]
```diff
@@ -7,13 +7,13 @@ excerpt: A guide to using container contracts to enforce device compatibility
 
 **Note** Container contracts are available for devices running Supervisor versions >= 10.6.17. All prior versions will not enforce any contracts.
 
-Container contracts are used to ensure the compatibility of a device to run an individual service of an application. For example, container contracts may be used to ensure that the device is running a specific version of [Supervisor][supervisor] or has a specific version of the [NVIDIA Tegra Linux Driver Package][l4t] (L4T).
+Container contracts are used to ensure the compatibility of a device to run an individual service of a fleet. For example, container contracts may be used to ensure that the device is running a specific version of [Supervisor][supervisor] or has a specific version of the [NVIDIA Tegra Linux Driver Package][l4t] (L4T).
```
Comment (chrisys) [inline pages/learn/develop/container-contracts.md:10]:
```suggestion
Container contracts are used to ensure the compatibility of a device when it comes to run a service. For example, container contracts may be used to ensure that the device is running a specific version of [Supervisor][supervisor] or has a specific version of the [NVIDIA Tegra Linux Driver Package][l4t] (L4T).
```

File: pages/learn/develop/container-contracts.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ requires:
...
 
-When deploying a release, if the contract requirements are not met, the release will fail, and all services on the device will remain on their current release. This default behavior can be overridden by including an [optional label](#optional-containers) in the application docker-compose.yml file. This optional label specifies which services with unmet requirements will be ignored, with all other services on the device updating to the new release. If there are any existing running services with unmet requirements, they will not be destroyed and continue running the prior release.
+When deploying a release, if the contract requirements are not met, the release will fail, and all services on the device will remain on their current release. This default behavior can be overridden by including an [optional label](#optional-containers) in the fleet docker-compose.yml file. This optional label specifies which services with unmet requirements will be ignored, with all other services on the device updating to the new release. If there are any existing running services with unmet requirements, they will not be destroyed and continue running the prior release.
```
Comment (chrisys) [inline pages/learn/develop/container-contracts.md:70]:
```suggestion
When deploying a release, if the contract requirements are not met, the release will fail, and all services on the device will remain on their current release. This default behavior can be overridden by including an [optional label](#optional-containers) in the docker-compose.yml file for the release. This optional label specifies which services with unmet requirements will be ignored, with all other services on the device updating to the new release. If there are any existing running services with unmet requirements, they will not be destroyed and continue running the prior release.
```

File: pages/learn/develop/container-contracts.md [hunk 3]
```diff
@@ -76,13 +76,13 @@ Could not move to new release: Some releases were rejected due to having unmet r
 Contracts: Services with unmet requirements: first-service
 ```
 
-**Note** For application fleets that contain a mixture of Supervisor and L4T versions, contracts may be used to ensure only compatible devices run specific services, as releases will only be deployed to those meeting the contract requirements.
```
Comment (chrisys) [inline pages/learn/develop/container-contracts.md:79]: This is a good one, what on earth is an 'application fleet' 😆

File: pages/learn/develop/container-contracts.md [hunk 4]
```diff
@@ -76,13 +76,13 @@ Could not move to new release: Some releases were rejected due to having unmet r
...
+**Note** For fleets that contain a mixture of Supervisor and L4T versions, contracts may be used to ensure only compatible devices run specific services, as releases will only be deployed to those meeting the contract requirements.
 
 ## Optional containers
 
-By default, when a container contract fails, none of the services are deployed to the device. However, in a multi-container application, it is possible to ignore those services that fail the contract requirements with the other services being deployed as normal. To do so, we make use of the `io.balena.features.optional: 1` [Supervisor label][labels] to indicate which services should be considered optional.
+By default, when a container contract fails, none of the services are deployed to the device. However, in a multi-container fleet, it is possible to ignore those services that fail the contract requirements with the other services being deployed as normal. To do so, we make use of the `io.balena.features.optional: 1` [Supervisor label][labels] to indicate which services should be considered optional.
```
Comment (chrisys) [inline pages/learn/develop/container-contracts.md:83]:
```suggestion
By default, when a container contract fails, none of the services are deployed to the device. However, in a multi-container release, it is possible to ignore those services that fail the contract requirements with the other services being deployed as normal. To do so, we make use of the `io.balena.features.optional: 1` [Supervisor label][labels] to indicate which services should be considered optional.
```

File: pages/learn/develop/container-contracts.md [hunk 5]
```diff
@@ -76,13 +76,13 @@ Could not move to new release: Some releases were rejected due to having unmet r
...
 
-In the `docker-compose.yml` file of the application, add the `io.balena.features.optional: 1` to the labels list for each service you wish to mark as optional. In the following example, even if the `first-service` container contract fails, the `second-service` service will still be deployed (assuming it doesn’t have any failing contracts of its own).
+In the `docker-compose.yml` file of the fleet, add the `io.balena.features.optional: 1` to the labels list for each service you wish to mark as optional. In the following example, even if the `first-service` container contract fails, the `second-service` service will still be deployed (assuming it doesn’t have any failing contracts of its own).
```
Comment (chrisys) [inline pages/learn/develop/container-contracts.md:85]:
```suggestion
In the `docker-compose.yml` file, add the `io.balena.features.optional: 1` to the labels list for each service you wish to mark as optional. In the following example, even if the `first-service` container contract fails, the `second-service` service will still be deployed (assuming it doesn’t have any failing contracts of its own).
```

File: pages/learn/develop/dockerfile.md [hunk 6]
```diff
@@ -5,7 +5,7 @@ excerpt: Use Dockerfiles to package your {{ $names.company.lower }} services and
 
 # Define a container
 
-{{ $names.company.upper }} uses [Docker][docker] containers to manage applications. You can use one or more containers to package your services with whichever environments and tools they need to run.
+{{ $names.company.upper }} uses [Docker][docker] containers to manage fleets. You can use one or more containers to package your services with whichever environments and tools they need to run.
```
Comment (chrisys) [inline pages/learn/develop/dockerfile.md:8]:
```suggestion
{{ $names.company.upper }} uses [Docker][docker] containers to manage deployment and updates. You can use one or more containers to package your services with whichever environments and tools they need to run.
```

File: pages/learn/develop/dockerfile.md [hunk 7]
```diff
@@ -27,13 +27,13 @@ Typically you will only need to use 4 instructions - [FROM][from], [RUN][run] an
...
 
 For details on other instructions, consult the official [Dockerfile documentation][dockerfile].
 
 ### Using Dockerfiles with {{ $names.company.lower }}
 
-To deploy a single-container application to {{ $names.company.lower }}, simply place a `Dockerfile` at the root of your repository. A `docker-compose.yml` file will be automatically generated, ensuring your container has host networking, is privileged, and has `lib/modules`, `/lib/firmware`, and `/run/dbus` bind mounted into the container. The default `docker-compose.yml` will look something like this:
+To deploy a single-container fleet to {{ $names.company.lower }}, simply place a `Dockerfile` at the root of your repository. A `docker-compose.yml` file will be automatically generated, ensuring your container has host networking, is privileged, and has `lib/modules`, `/lib/firmware`, and `/run/dbus` bind mounted into the container. The default `docker-compose.yml` will look something like this:
```
Comment (chrisys) [inline pages/learn/develop/dockerfile.md:36]:
```suggestion
To deploy a single-container release to {{ $names.company.lower }}, simply place a `Dockerfile` at the root of your repository. A `docker-compose.yml` file will be automatically generated, ensuring your container has host networking, is privileged, and has `lib/modules`, `/lib/firmware`, and `/run/dbus` bind mounted into the container. The default `docker-compose.yml` will look something like this:
```

File: pages/learn/develop/dockerfile.md [hunk 8]
```diff
@@ -59,7 +59,7 @@ services:
       io.balena.features.balena-api: '1'
 ```
 
-Applications with multiple containers should include a `Dockerfile` or `package.json` in each service directory. A `docker-compose.yml` file will need to be defined at the root of the repository, as discussed in our [multicontainer documentation][multicontainer].
+Fleets with multiple services should include a `Dockerfile` or `package.json` in each service directory. A `docker-compose.yml` file will need to be defined at the root of the repository, as discussed in our [multicontainer documentation][multicontainer].
```
Comment (chrisys) [inline pages/learn/develop/dockerfile.md:62]:
```suggestion
Releases with multiple services should include a `Dockerfile` or `package.json` in each service directory. A `docker-compose.yml` file will need to be defined at the root of the repository, as discussed in our [multicontainer documentation][multicontainer].
```

File: pages/learn/develop/dockerfile.md [hunk 9]
```diff
@@ -86,15 +86,15 @@ CMD ["node", "/usr/src/app/main.js"]
 This template will build and deploy a Node.js project for any of the devices supported by {{ $names.company.lower }}, regardless of whether the device architecture is [ARM][ARM-link] or [x86][x86-link].
 In this example, you can see the build variable `%%{{ $names.company.allCaps }}_MACHINE_NAME%%`. This will be replaced by the machine name (i.e.: `raspberry-pi`) at build time. See below for a list of machine names.
 
-The machine name is inferred from the device type of the application you are pushing to. So if you have an Intel Edison application, the machine name will be `intel-edison` and an `i386` architecture base image will be built.
+The machine name is inferred from the device type of the fleet you are pushing to. So if you have an Intel Edison fleet, the machine name will be `intel-edison` and an `i386` architecture base image will be built.
```
Comment (chrisys) [inline pages/learn/develop/dockerfile.md:89]:
```suggestion
The machine name is inferred from the device type of the fleet you are deploying on. So if you have a NanoPi Neo Air fleet, the machine name will be `nanopi-neo-air` and an `armv7hf` architecture base image will be built.
```

File: pages/learn/develop/dockerfile.md [hunk 10]
```diff
@@ -106,36 +106,27 @@ Here are the supported machine names and architectures:
 
 There are cases when you would need a higher granularity of control when specifying build instructions for different devices and architectures than a single Dockerfile template can provide. An example of this would be when different configuration or installation files are required for each architecture or device.
 
-When deploying an application, the balenaCloud build servers or the balena CLI tool (depending on the deployment method used) look at all available Dockerfiles and build the appropriate image using the following order of preference:
+When deploying a fleet, the balenaCloud build servers or the balena CLI tool (depending on the deployment method used) look at all available Dockerfiles and build the appropriate image using the following order of preference:
```
Comment (chrisys) [inline pages/learn/develop/dockerfile.md:109]:
```suggestion
When creating a release, the balenaCloud build servers or the balena CLI tool (depending on the deployment method used) look at all available Dockerfiles and build the appropriate image using the following order of preference:
```

File: pages/learn/develop/dockerfile.md [hunk 11]
```diff
@@ -106,36 +106,27 @@ Here are the supported machine names and architectures:
...
 
 * Dockerfile.\<device-type>
 * Dockerfile.\<arch>
 * Dockerfile.template
 
-As an example, let's say you have two Dockerfiles available, `Dockerfile.raspberrypi3` and `Dockerfile.template`. Whenever you publish the application to balenaCloud, if the `device-type` is a Raspberry Pi 3, `Dockerfile.raspberrypi3` will be selected as an exact match and for all other devices the builder will automatically select `Dockerfile.template`.
+As an example, let's say you have two Dockerfiles available, `Dockerfile.raspberrypi3` and `Dockerfile.template`. Whenever you publish the application to balenaCloud, if the fleet `device-type` is a Raspberry Pi 3, `Dockerfile.raspberrypi3` will be selected as an exact match and for all other devices the builder will automatically select `Dockerfile.template`.
```
Comment (chrisys) [inline pages/learn/develop/dockerfile.md:115]:
```suggestion
As an example, let's say you have two Dockerfiles available, `Dockerfile.raspberrypi3` and `Dockerfile.template`. Whenever you publish the application to balenaCloud, if the fleet `device-type` is Raspberry Pi 3, `Dockerfile.raspberrypi3` will be selected as an exact match and for all other devices the builder will automatically select `Dockerfile.template`.
```

File: pages/learn/develop/dockerfile.md [hunk 12]
```diff
@@ -106,36 +106,27 @@ Here are the supported machine names and architectures:
...
-generates a [container][container] for the environment your device operates in,
-deploys your code to it and runs `npm install` to resolve [npm][npm]
-dependencies, reporting progress to your terminal as it goes.
+When you push your code to your fleet, the build server generates a [container][container] for the environment your device operates in, deploys your code to it and runs `npm install` to resolve [npm][npm] dependencies, reporting progress to your terminal as it goes.
 
-If the build executes successfully the container is shipped over to your device
-where the supervisor runs it in place of any previously running containers,
-using `npm start` to execute your code (note that if no start script is
-specified, it defaults to running `node server.js`.)
+If the build executes successfully the container is shipped over to your device where the supervisor runs it in place of any previously running containers, using `npm start` to execute your code (note that if no start script is specified, it defaults to running `node server.js`.)
```
Comment (chrisys) [inline pages/learn/develop/dockerfile.md:125]:
```suggestion
If the build executes successfully the release is deployed to your device where the supervisor runs it in place of any previously running containers, using `npm start` to execute your code (note that if no start script is specified, it defaults to running `node server.js`.)
```

File: pages/learn/develop/hardware/i2c-and-spi.md [hunk 13]
```diff
@@ -84,7 +84,7 @@ This can be done in two ways:
 ```
 BALENA_HOST_CONFIG_dtoverlay = pi3-miniuart-bt
 ```
-If you can't find the where to add this configuration go to this page on your dashboard: dashboard.balena-cloud.com/apps/`APP_ID`/config but replace `APP_ID` with the number of your application.
+If you can't find the where to add this configuration go to this page on your dashboard: dashboard.balena-cloud.com/apps/`APP_ID`/config but replace `APP_ID` with the number of your fleet.
```
Comment (chrisys) [inline pages/learn/develop/hardware/i2c-and-spi.md:87]: This is crazy! We're suggesting that if they can't find the page to manually construct a URL and visit that.

```suggestion
```

File: pages/learn/develop/hardware/i2c-and-spi.md [hunk 14]
```diff
@@ -153,7 +153,7 @@ so won't work with the 16M GPU split.
 ### Customizing config.txt
 These are some tips and tricks for customizing your raspberry pi. Most of them require changing settings in the `config.txt` file on the SD cards `boot` partition. See [here](/configuration/advanced/) for more details.
 
-You can also set all of these variables remotely in the Device Configuration (for a single device) or Fleet Configuration (for all devices within an application) menu. If the setting in `config.txt` is `variable=value`, you can achieve the same settings by adding a configuration variable with `BALENA_HOST_CONFIG_variable` set to the value `value`. For example:
+You can also set all of these variables remotely in the Device Configuration (for a single device) or Fleet Configuration (for all devices within an fleet) menu. If the setting in `config.txt` is `variable=value`, you can achieve the same settings by adding a configuration variable with `BALENA_HOST_CONFIG_variable` set to the value `value`. For example:
```
Comment (chrisys) [inline pages/learn/develop/hardware/i2c-and-spi.md:156]:
```suggestion
You can also set all of these variables remotely in the Device Configuration (for a single device) or Fleet Configuration (for all devices within a fleet) menu. If the setting in `config.txt` is `variable=value`, you can achieve the same settings by adding a configuration variable with `BALENA_HOST_CONFIG_variable` set to the value `value`. For example:
```

File: pages/learn/develop/hardware/i2c-and-spi.md [hunk 15]
```diff
@@ -272,7 +272,7 @@ The Edison needs a kernel module to be loaded to trigger the UBS HOST mode. This
 Your Edison will need to be powered externally for the USB host mode to be active - Either through the DC jack on the Arduino board or through the battery connector on the smaller Intel carrier board.
 
 ##### Software Pre-requisites:
-The following code needs to be placed at the start before any device operations are run in your application container.
+The following code needs to be placed at the start before any device operations run in the services.
```
Comment (chrisys) [inline pages/learn/develop/hardware/i2c-and-spi.md:275]:
```suggestion
The following code needs to be placed at the start before any device operations run in the container.
```

File: pages/learn/develop/multicontainer.md [hunk 16]
```diff
@@ -5,21 +5,19 @@ excerpt: A guide to running multiple containers with Docker Compose and {{ $name
...
+As your apps grow more complex, you may find significant benefit in running some services in separate containers. Splitting your app into multiple containers allows you to better isolate and maintain key services, providing a more modular and secure approach to fleet management. Each service can be packaged with the operating environment and tools it specifically needs to run, and each service can be limited to the minimum system resources necessary to perform its task. The benefits of multicontainer fleets compound as the complexity of the app grows. With multicontainers, each service is updated independently. Hence, larger apps can be developed and maintained by separate teams, each free to work in a way that best supports their service.
```
Comment (chrisys) [inline pages/learn/develop/multicontainer.md:8]:
```suggestion
As your apps grow more complex, you may find significant benefit in running some services in separate containers. Splitting your app into multiple containers allows you to better isolate and maintain key services, providing a more modular and secure approach to fleet management. Each service can be packaged with the operating environment and tools it specifically needs to run, and each service can be limited to the minimum system resources necessary to perform its task. The benefits of multicontainer fleets compound as the complexity of the app grows. With multicontainer, each service is updated independently. Hence, larger apps can be developed and maintained by separate teams, each free to work in a way that best supports their service.
```

File: pages/learn/develop/multicontainer.md [hunk 17]
```diff
@@ -5,21 +5,19 @@ excerpt: A guide to running multiple containers with Docker Compose and {{ $name
...
-
 ## docker-compose.yml file
 
 The multicontainer functionality provided by {{ $names.company.lower }} is built around the **[Docker Compose][docker-compose]** file format. The {{ $names.company.lower }} device supervisor implements a subset of the [Compose v2.1 feature set][compose-features]. You can find a full list of supported and known unsupported features in our [device supervisor reference docs][compose-support].
 
-At the root of your multicontainer application, you'll use a `docker-compose.yml` file to specify the configuration of your containers. The `docker-compose.yml` defines the services you'll be building, as well as how the services interact with each other and the host OS.
+At the root of your multicontainer fleet, you'll use a `docker-compose.yml` file to specify the configuration of your containers. The `docker-compose.yml` defines the services you'll be building, as well as how the services interact with each other and the host OS.
```
Comment (chrisys) [inline pages/learn/develop/multicontainer.md:18]:
```suggestion
At the root of your multicontainer release, you'll use a `docker-compose.yml` file to specify the configuration of your containers. The `docker-compose.yml` defines the services you'll be building, as well as how the services interact with each other and the host OS.
```

File: pages/learn/develop/multicontainer.md [hunk 18]
```diff
@@ -5,21 +5,19 @@ excerpt: A guide to running multiple containers with Docker Compose and {{ $name
...
 
-Here's an example `docker-compose.yml` for a [simple multicontainer application][simple-app], composed of a static site server, a websocket server, and a proxy:
+Here's an example `docker-compose.yml` for a [simple multicontainer fleet][simple-app], composed of a static site server, a websocket server, and a proxy:
```
Comment (chrisys) [inline pages/learn/develop/multicontainer.md:20]:
```suggestion
Here's an example `docker-compose.yml` for a [simple multicontainer release][simple-app], composed of a static site server, a websocket server, and a proxy:
```

File: pages/learn/develop/runtime.md [hunk 19]
```diff
@@ -6,7 +6,7 @@ thumbnail: /img/common/device/running-webterminal-session.png
 
 # Communicate outside the container
 
-For many applications, the code running in your container will need some way to communicate with outside services and devices, whether this means the host OS, the device supervisor, the network, or internal and external storage. Below,  you'll find different methods for external communication, as well as some tips and tricks to keep in mind.
+For many apps, the code running in your container will need some way to communicate with outside services and devices, whether this means the host OS, the device supervisor, the network, or internal and external storage. Below, you'll find different methods for external communication, as well as some tips and tricks to keep in mind.
```
Comment (chrisys) [inline pages/learn/develop/runtime.md:9]:
```suggestion
In many situations, the code running in your container will need some way to communicate with outside services and devices, whether this means the host OS, the device supervisor, the network, or internal and external storage. Below, you'll find different methods for external communication, as well as some tips and tricks to keep in mind.
```

File: pages/learn/develop/runtime.md [hunk 20]
```diff
@@ -189,7 +189,7 @@ Alternatively, it is possible to reboot the device via the D-Bus interface as de
 
 ### Writing to logs on the Dashboard
 
-Anything written from the application to `stdout` and `stderr` should appear on the device's dashboard logs. Have a look at some of our [example projects][projects-github] on GitHub to get an idea of how to do this.
+Anything written from the app to `stdout` and `stderr` should appear on the device's dashboard logs. Have a look at some of our [example projects][projects-github] on GitHub to get an idea of how to do this.
```
Comment (chrisys) [inline pages/learn/develop/runtime.md:192]:
```suggestion
Anything written to `stdout` and `stderr` should appear on the device's dashboard logs. Have a look at some of our [example projects][projects-github] on GitHub to get an idea of how to do this.
```

File: pages/learn/develop/runtime.md [hunk 21]
```diff
@@ -198,17 +198,17 @@ Anything written from the application to `stdout` and `stderr` should appear on
 * Host mode allows a service to use all host network interfaces.
 * Bridge mode uses a user-defined bridge network interface, to which service containers are connected.
 
-Any service that uses host networking does not have to define ports for traffic ingress, and a service can bind to all interfaces of the host. Single container applications always use host networking.
+Any service that uses host networking does not have to define ports for traffic ingress, and a service can bind to all interfaces of the host. Single container fleets always use host networking.
```
Comment (chrisys) [inline pages/learn/develop/runtime.md:201]:
```suggestion
Any service that uses host networking does not have to define ports for traffic ingress, and a service can bind to all interfaces of the host. Single container releases always use host networking.
```

File: pages/learn/develop/runtime.md [hunk 22]
```diff
@@ -198,17 +198,17 @@ Anything written from the application to `stdout` and `stderr` should appear on
...
 
-In contrast to host networking, bridge networks isolate all services from the host, requiring services to explicitly define open ports to allow traffic from the host to be passed to them (all outgoing traffic is permitted). By default, multicontainer applications use a bridge network.
+In contrast to host networking, bridge networks isolate all services from the host, requiring services to explicitly define open ports to allow traffic from the host to be passed to them (all outgoing traffic is permitted). By default, multicontainer fleets use a bridge network.
```
Comment (chrisys) [inline pages/learn/develop/runtime.md:203]:
```suggestion
In contrast to host networking, bridge networks isolate all services from the host, requiring services to explicitly define open ports to allow traffic from the host to be passed to them (all outgoing traffic is permitted). By default, multicontainer releases use a bridge network.
```

File: pages/learn/develop/runtime.md [hunk 23]
```diff
@@ -198,17 +198,17 @@ Anything written from the application to `stdout` and `stderr` should appear on
...
 
-### Single container applications
+### Single container fleets
```
Comment (chrisys) [inline pages/learn/develop/runtime.md:205]:
```suggestion
### Single container
```

File: pages/learn/develop/runtime.md [hunk 24]
```diff
@@ -198,17 +198,17 @@ Anything written from the application to `stdout` and `stderr` should appear on
...
 
-Single container applications always use host networking, allowing them to bind to any of the host's network interfaces. If security and sandboxing are required for either privilege level or to ensure self-contained networking, then a multicontainer application should be used, even if only a single service is required.
+Single container fleets always use host networking, allowing them to bind to any of the host's network interfaces. If security and sandboxing are required for either privilege level or to ensure self-contained networking, then a multicontainer fleet should be used, even if only a single service is required.
```
Comment (chrisys) [inline pages/learn/develop/runtime.md:207]:
```suggestion
Single container releases always use host networking, allowing them to bind to any of the host's network interfaces. If security and sandboxing are required for either privilege level or to ensure self-contained networking, then a multicontainer release should be created, even if only a single service is required.
```

File: pages/learn/develop/runtime.md [hunk 25]
```diff
@@ -198,17 +198,17 @@ Anything written from the application to `stdout` and `stderr` should appear on
...
 
-### Multicontainer applications
+### Multicontainer fleets
```
Comment (chrisys) [inline pages/learn/develop/runtime.md:209]:
```suggestion
### Multicontainer
```

File: pages/learn/develop/runtime.md [hunk 26]
```diff
@@ -198,17 +198,17 @@ Anything written from the application to `stdout` and `stderr` should appear on
...
 
-Multicontainer applications use a user-defined bridge network by default. No ports are exposed to the host and must be explicitly enabled through the `ports` [keyword][network-ports]. Services on the same bridge network have access to all other services' ports.
+Multicontainer fleets use a user-defined bridge network by default. No ports are exposed to the host and must be explicitly enabled through the `ports` [keyword][network-ports]. Services on the same bridge network have access to all other services' ports.
```
Comment (chrisys) [inline pages/learn/develop/runtime.md:211]:
```suggestion
Multicontainer releases use a user-defined bridge network by default. No ports are exposed to the host and must be explicitly enabled through the `ports` [keyword][network-ports]. Services on the same bridge network have access to all other services' ports.
```

File: pages/learn/develop/runtime.md [hunk 27]
```diff
@@ -234,7 +234,7 @@ services:
 
 __Note:__ Exposing ports via the expose keyword is optional and a way of documenting which ports are used, but does not map or open any ports. By default, services on the same bridge network have access to all other services' ports.
 
-For multicontainer applications, setting the service `network_mode` to `host` in `docker-compose.yml` allows the container to share the same network namespace as the host OS.
+For multicontainer fleets, setting the service `network_mode` to `host` in `docker-compose.yml` allows the container to share the same network namespace as the host OS.
```
Comment (chrisys) [inline pages/learn/develop/runtime.md:237]:
```suggestion
For multicontainer releases, setting the service `network_mode` to `host` in `docker-compose.yml` allows the container to share the same network namespace as the host OS.
```

File: pages/learn/develop/container-contracts.md (hunk 3 above)
Comment (vipulgupta2048) [reply pages/learn/develop/container-contracts.md:79]: I sure don't wanna know them.

**Comment by vipulgupta2048** [comment]
Ready to go, pending rebase

**Comment by vipulgupta2048** [comment]
@balena-ci rebase

**Review by chrisys — APPROVED** [review]

**Pull request merged**